### PR TITLE
New styles

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/shared/components/text-input/text-input.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<ng-container class="form-group">
+  <input type="text"
+         [id]="name"
```
Comment (Rementh) [inline src/app/shared/components/text-input/text-input.component.html]: Is that used somwhere?

File: src/app/shared/components/text-input/text-input.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  styleUrls: ['./text-input.component.scss'],
+  viewProviders: [{provide: ControlContainer, useExisting: NgForm}]
+})
+export class TextInputComponent implements OnInit {
+
+  @Input() name: string;
+  @Input() field: string;
+
+  constructor() {
+  }
```
Comment (Rementh) [inline src/app/shared/components/text-input/text-input.component.ts]: Delete if not needed

Comment (ewelina-slepko) [reply src/app/shared/components/text-input/text-input.component.ts]: ok

File: src/app/authentication/sign-in/sign-in.component.html [hunk 3]
```diff
@@ -1,26 +1,14 @@
-<form #signInForm="ngForm" (ngSubmit)="signInWithEmailAndPassword(user)">
-  <label></label>
-  <input id="email"
-         name="email"
-         #email="ngModel"
-         placeholder="e-mail"
-         [(ngModel)]="user.email"
-         required>
+<h3 class="header">It's great to see you again</h3>
```
[the remark this answers is not in the document]
Comment (ewelina-slepko) [reply src/app/authentication/sign-in/sign-in.component.html]: ok

File: src/app/shared/components/text-input/text-input.component.html (hunk 1 above)
Comment (ewelina-slepko) [reply src/app/shared/components/text-input/text-input.component.html]: ok

**Comment by ewelina-slepko** [comment]
Done

**Pull request merged**